src/main/resources/templates/datewarehouse/etl-jobs-config.json

```diff
@@ -299,18 +744,47 @@
 		"sourcePath" : "log.sagebase.org/aclSnapshots/",
 		"tableDescriptor": {
 			"name": "aclsnapshots",
-			"columns": {
-				"change_timestamp": "timestamp",
-				"change_type": "string",
-				"snapshot_timestamp": "timestamp",
-				"owner_id": "bigint",
-				"owner_type": "string",
-				"created_on": "timestamp",
-				"resource_access": "array<struct<principalId:bigint,accessType:array<string>>>"
-			},
-			"partitionKeys": {
-				"snapshot_date": "date"
+			"description" : "This table contains snapshots of access-control-list. Snapshots are taken when acl is created, updated or deleted. Note: Snapshots are also taken periodically and independently of the changes. The snapshot_timestamp records when the snapshot was taken.",
+			"columns":[{
+				"name": "change_timestamp",
+				"type" : "timestamp",
+				"comment" : "The time when the user created/updated/deleted the acl."
+			},
+			{
+				"name": "change_type",
+				"type" : "string",
+				"comment" : "The type of changed that occurred on acl like CREATE, UPDATE, DELETE."
+			},
+			{
+				"name": "snapshot_timestamp",
+				"type" : "timestamp",
+				"comment" : "The time when the snapshot was taken(It's usually shortly after the change happened)."
 			},
+			{
+				"name": "owner_id",
+				"type" : "bigint",
+				"comment" : "The unique identifier of the Synapse object that the acl is affecting like node-id, team-id..."
```

Comment:
'node-id' is not a 'unique identifier'.  I would just drop everythign after "...is affecting." Here's a better rewrite:

"The unique identifier of the Synapse object to which the acl is applied."